src/lib/viewers/BaseViewer.js

```diff
@@ -1081,6 +1081,30 @@ class BaseViewer extends EventEmitter {
         this.annotator.toggleAnnotationMode(AnnotationMode.NONE);
     }
 
+    /**
+     * Handler for annotation mode change
+     * 1. Set annotationsControls mode
+     * 2. If discoverability FF is on, add mode classes to container
+     *
+     * @param {AnnotationMode} mode Next annotation mode
+     */
+    handleAnnotationModeChange = mode => {
```

Comment:
Maybe instead of `handle*` since by convention that tends to be an event handler, `processAnnotationModeChange`?